.github/workflows/frontendBuild.yml

```diff
@@ -0,0 +1,36 @@
+name: Frontend Lint and tests
+
+on:
+  push:
+    branches: [trunk]
+  pull_request:
+    branches: [trunk]
+
+jobs:
+  lint_build_and_run_tests:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use Node 16
```

Comment:
I noticed we don't currently specify which node version to use for the project in the README, can we add that?

Reply:
Not suggesting any change here. It's also worth reading about this blog post which highlights the importance of testing against multiple node versions: https://nodejs.medium.com/choosing-the-node-js-versions-for-your-ci-tests-hint-use-lts-89b67f68d7ca

Reply:
woah breaking changes every 6 months... Thanks for sharing this! I ended up updating the build file anyway to see how it works. [d156996](https://github.com/jramnani/GildedRoseJavaLLC/pull/35/commits/d156996131074917a793fbc71d3c07a2a1f60a0c)

Reply:
added node 18 to matrix as well since this was build using that version. README updated as well [5f997a1](https://github.com/jramnani/GildedRoseJavaLLC/pull/35/commits/5f997a1788f70df19e1fb10551ca4b2f3ed9467c)